README.md

```diff
@@ -3,63 +3,70 @@
 [![test](https://github.com/blocoio/android-template/workflows/test/badge.svg?branch=master)](https://github.com/blocoio/android-template/actions?query=workflow%3Atest+branch%3Amaster)
 [![lint](https://github.com/blocoio/android-template/workflows/lint/badge.svg?branch=master)](https://github.com/blocoio/android-template/actions?query=workflow%3Alint+branch%3Amaster)
 
-## Clean architecture with 3 layers
+The Goal of this project is to have a starting point when starting a new project that follows the best development practices, this project is our interpretation and in-house solution to meet the official [architecture](https://developer.android.com/topic/architecture) guidelines provided by Google.
+This template is also inspired by Google [NowInAndroid](https://github.com/android/nowinandroid) *“is a fully functional Android app built entirely with Kotlin and Jetpack Compose. It follows Android design and development best practices…”.*  
+
+This version brings [Modularization](https://developer.android.com/topic/modularization). With that also came:
+
+- [Version catalogs](https://docs.gradle.org/current/userguide/platforms.html) are a type safe list of dependencies
+  generated by Gradle during sync. It’s a central place to declare all your
+  dependencies and is available to all the modules in a project.
+- Use [convention plugins](https://docs.gradle.org/current/samples/sample_convention_plugins.html) to share build logic between modules.
```

Comment:
This is the general readme, I feel it shouldn't mention versions, just describe how it is at the moment.

Also, I think this section should come after the architecture since it's an implementation detail.